cirq/google/calibration/phased_fsim.py

```diff
@@ -185,6 +185,25 @@ class PhasedFSimCalibrationResult:
     gate: Gate
     options: PhasedFSimCalibrationOptions
 
+    def override(self, parameters: PhasedFSimCharacterization) -> 'PhasedFSimCalibrationResult':
+        """Creates the new results with certain parameters overridden for all characterizations.
+
```

Comment:
Add "This functionality can be used to zero-out the corrected angles and do the analysis on remaining errors."

Reply:
Done.